README.md

```diff
@@ -8,11 +8,11 @@ Install the SDK as a module: `npm i @kissflow/lowcode-client-sdk` Then import
 into your project:
 
 ```js
-import KFSDK from "@kissflow/lowcode-client-sdk";
-let kf;
-(async function () {
-	kf = await KFSDK.initialize();
-})();
+import KFSDK from '@kissflow/lowcode-client-sdk'
+let kf
+;(async function () {
```

Comment:
check

Reply:
Its a self invoking function bro... The change were not made by me, it was made by prettier (I added prettier and its sensible default rules)... 

**Prettier usually formats the files with sensibles, but if the semicolon and parenthesis are causing confusion to the reader of the docs, I shall ask prettier to avoid README.md file.**